src/pages/Integrations/Create/CreateWizard.tsx

```diff
@@ -34,7 +34,9 @@ export const CreateWizard: React.FunctionComponent<CreateWizardProps> = ({
   return isOpen ? (
     <FormRenderer
       schema={schema(category)}
-      FormTemplate={Pf4FormTemplate}
+      FormTemplate={(props) => (
+        <Pf4FormTemplate {...props} showFormControls={false} />
```

Comment:
This is a bug I noticed where opening the `Add Integration` wizard causes submit buttons to show in the UI behind the modal.